examples/remix/README.md

```diff
@@ -0,0 +1,186 @@
+  presets: [
+    ['@babel/preset-env', { targets: { node: 'current' } }],
+    ['@babel/preset-react', { runtime: 'automatic' }],
+  ],
+};
+```
+
+Next, install your favorite testing libraries. This example uses `react-testing-library`, at version 12 specifically ([version 13 wouldn't work](https://stackoverflow.com/questions/71713405/cannot-find-module-react-dom-client-from-node-modules-testing-library-react)). Optionally, you can install its companion [`jest-dom`](https://testing-library.com/docs/ecosystem-jest-dom/) package to add some useful matchers to Jest.
+
+```bash
+npm install --save-dev @testing-library/react@12 @testing-library/jest-dom
+```
+
+## Install & configure Jest Preview
+
+Install Jest Preview
+
+```bash
+npm install --save-dev jest-preview
+```
+
+Enable Jest Preview inside `jest.config.js`. The config options are largely up to you, but remember to remove/add the highlighted options as followed so Jest Preview could work properly.
+
+```diff
+module.exports = {
+  collectCoverageFrom: [
+    '**/*.{js,jsx,ts,tsx}',
+    '!**/*.d.ts',
+    '!**/node_modules/**',
+  ],
+  moduleNameMapper: {
+-   // Handle CSS imports (with CSS modules)
+-   // https://jestjs.io/docs/webpack#mocking-css-modules
+-   '^.+\\.module\\.(css|sass|scss)$': 'identity-obj-proxy',
+-
+-   // Handle CSS imports (without CSS modules)
+-   '^.+\\.(css|sass|scss)$': '<rootDir>/__mocks__/styleMock.js',
+-
+-   // Handle image imports
+-   // https://jestjs.io/docs/webpack#handling-static-assets
+-   '^.+\\.(png|jpg|jpeg|gif|webp|avif|ico|bmp|svg)$/i': `<rootDir>/__mocks__/fileMock.js`,
++   // Handle absolute imports in Remix
++   '~/(.*)': '<rootDir>/app/$1',
+  },
++ setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
+  testPathIgnorePatterns: [
+    '<rootDir>/node_modules/',
+    '<rootDir>/.cache/',
+    '<rootDir>/build/',
+  ],
++ testEnvironment: 'jsdom',
+  transform: {
+    // Use babel-jest to transpile tests
+    // https://jestjs.io/docs/configuration#transform-objectstring-pathtotransformer--pathtotransformer-object
+    '^.+\\.(js|jsx|ts|tsx)$': 'babel-jest',
++   '^.+\\.(css|scss|sass)$': 'jest-preview/transforms/css',
++   '^(?!.*\\.(js|jsx|mjs|cjs|ts|tsx|css|json)$)': 'jest-preview/transforms/file',
+  },
+  transformIgnorePatterns: [
+    '/node_modules/',
+-   '^.+\\.module\\.(css|sass|scss)$',
+  ],
+};
+```
+
+Configure Jest Preview inside `jest.setup.js` (or any setup files specified in your `setupFilesAfterEnv` config), so Jest Preview knows which global CSS file to load. You can even set `autoPreview` to `true` so your failed test gets a preview automatically! 🤯
+
+```js
+import { jestPreviewConfigure } from 'jest-preview';
+
+jestPreviewConfigure({
+  // An array of relative paths from the root of your project
+  externalCss: [
+    'app/styles/global.css',
+  ],
+  // (Optional) Enable autoPreview so Jest Preview runs automatically
+  // whenever your test fails, without you having to do anything!
+  // autoPreview: true,
```

Comment:
We can enable it by default if there is no problem yet. It will be the default option soon anyway.